app/views/nic/_base_form.html.erb

```diff
@@ -28,25 +30,47 @@
                  :size => "col-md-8", :label_size => "col-md-3" } %>
-  <%= select_f f, :subnet_id, accessible_resource(f.object, :subnet), :id, :to_label,
-               { :include_blank => accessible_resource(f.object, :subnet).any? ? true : _("No subnets")},
-               { :disabled => accessible_resource(f.object, :subnet).empty? ? true : false,
+  <%= select_f f, :subnet_id, accessible_resource(f.object, :"Subnet::Ipv4"), :id, :to_label,
+               { :include_blank => accessible_resource(f.object, :"Subnet::Ipv4").any? ? true : _("No subnets")},
+               { :disabled => accessible_resource(f.object, :"Subnet::Ipv4").empty? ? true : false,
+                 :label => _("IPv4 Subnet"),
                  :help_inline => :indicator,
                  :class => 'interface_subnet', :'data-url' => freeip_subnets_path,
                  :size => "col-md-8", :label_size => "col-md-3" } %>
+  <%= select_f f, :subnet6_id, accessible_resource(f.object, :"Subnet::Ipv6"), :id, :to_label,
+               { :include_blank => accessible_resource(f.object, :"Subnet::Ipv6").any? ? true : _("No subnets")},
+               { :disabled => accessible_resource(f.object, :"Subnet::Ipv6").empty? ? true : false,
+                 :help_inline => :indicator,
+                 :label => _("IPv6 Subnet"),
+                 :class => 'interface_subnet6', :'data-url' => freeip_subnets_path,
+                 :size => "col-md-8", :label_size => "col-md-3" } %>
 <% end %>
 <%= text_f f, :ip,
               :class => :interface_ip,
+              :label => _("IPv4 address"),
               :autocomplete => 'off',
               :help_block => link_to(_("Suggest new"), '#', :class => 'suggest_new_ip'),
               :help_inline => popover('',
-                  _("An IP address will be auto-suggested if you have a DHCP-enabled Smart Proxy on the subnet selected above.<br/><br/>The IP address can be left blank when:<br/><ul><li>provisioning tokens are enabled</li><li>the domain does not manage DNS</li><li>the subnet does not manage reverse DNS</li><li>and the subnet does not manage DHCP reservations</li></ul>"),
+                  _("An IP address will be auto-suggested if you have enabled IPAM on the IPv4 subnet selected above.<br/><br/>The IP address can be left blank when:<br/><ul><li>provisioning tokens are enabled</li><li>the domain does not manage DNS</li><li>the subnet does not manage reverse DNS</li><li>and the subnet does not manage DHCP reservations</li></ul>"),
                   :title => _("IP address auto-suggest"),
                   :rel => 'popover-modal',
                   :'data-placement' => "top"
                 ).html_safe,
                 :size => "col-md-8", :label_size => "col-md-3" %>
 
+<%= text_f f, :ip6,
+              :label => _("IPv6 address"),
+              :class => :interface_ip6,
+              :autocomplete => 'off',
+              :help_block => link_to(_("Suggest new"), '#', :class => 'suggest_new_ip6'),
```

Comment:
This `Suggest new` should only be shown when that's actually possible: which is when `subnet.ipam` is `Internal DB`. Otherwise one can click w/o any feedback and it's confusing